src/fava/core/attributes.py

```diff
@@ -71,6 +73,11 @@ def load_file(self) -> None:
         currency_ranker = ExponentialDecayRanker()
         payee_ranker = ExponentialDecayRanker()
 
+        self.closing_dates = {
+            account: self.ledger.accounts[account].close_date
```

Comment:
sorry, I had forgotten about FavaLedger.accounts - since we have it already, let's just pass that to the frontend (maybe as account_details) and avoid the work here

Reply:
Alright. Then I think I need a pointer how to implement correct serialization for `AccountData` - the naive way didn't work 😉